snapcraft/models/project.py

```diff
@@ -661,12 +1033,12 @@ class Architecture(models.CraftBaseModel, extra="forbid"):
 
     build_on: str | UniqueList[str] = pydantic.Field(
         description="The architectures on which the snap can be built.",
-        examples=["build-on: amd64, riscv64"],
+        examples=["amd64, riscv64"],
```

Comment:
There are 4 occurrences of this (build-on and build-for in architectures and platforms)..

```suggestion
        examples=["[amd64, riscv64]"],
```

Reply:
Thanks for catching this!

https://github.com/canonical/snapcraft/pull/5421/commits/f42b363320603021670ce9de364ee1644ff80508